docs/concepts/configuration/assign-pod-node.md

```diff
@@ -139,6 +139,8 @@ If you specify multiple `matchExpressions` associated with `nodeSelectorTerms`,
 
 If you remove or change the label of the node where the pod is scheduled, the pod won't be removed. In other words, the affinity selection works only at the time of scheduling the pod.
 
+The `weight` field in `preferredDuringSchedulingIgnoredDuringExecution` is in the range 1-100. The scheduler will prefer to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights, i.e. for each node that meets all of the scheduling requirements (resource request, RequiredDuringScheduling affinity expressions, etc.), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding MatchExpressions; the node(s) with the highest sum are the most preferred.
```

Comment:
This is an amazing update, thank you. One request re: wording though - could you replace the `i.e.` with `for example` - we’re trying to not use latin abbreviations to make it easier for readers not familiar with idiomatic english

Reply:
If there is another commit, I'd like to suggest wrapping lines at 80 characters' boundary.
I'm suggesting this because GitHub.com doesn't provide a nice interface to highlight changes made or to add comments if lines are too long. Maybe I'm old-fashioned on this ...

Reply:
@heckj @tengqm Thanks for pointing out that. Really helpful for other readers and me. I've made updates.